api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd

```diff
@@ -26,8 +26,8 @@
 		(filter*),
 		(filter-mapping*),
 		(messages*),
-		(mappingFiles?)
-		(packagesWithMappedClasses?)
+		(mappingFiles?),
+		(packagesWithMappedClasses?),
```

Comment:
And the file too?

Reply:
they were showing an error and thought they were the reason my build was failing, can I make the change and make the pull request again